Mathlib/NumberTheory/Divisors.lean

```diff
@@ -3,9 +3,10 @@ Copyright (c) 2020 Aaron Anderson. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Aaron Anderson
 -/
-import Mathlib.Algebra.GroupPower.Order
 import Mathlib.Algebra.Order.BigOperators.Group.Finset
+import Mathlib.Algebra.Order.Ring.Basic
 import Mathlib.Data.Nat.Factors
+import Mathlib.Order.GameAdd
```

Comment:
This is pretty funny. The import is not actually required, but minimize_imports thinks it is unless you change
```diff
-  rw [mem_divisorsAntidiagonal, mem_divisorsAntidiagonal, mul_comm, Prod.swap]
+  rw [mem_divisorsAntidiagonal, mem_divisorsAntidiagonal, mul_comm, Prod.fst_swap, Prod.snd_swap]
```
because `Prod.swap.eq_1` ends up being generated lazily in `Mathlib.Order.GameAdd`.

So:
```suggestion
```

(In fact, `Mathlib.Algebra.Order.Ring.Basic` can be removed easily by replacing `pow_lt_pow_iff_right` → `Nat.pow_lt_pow_iff_right`, but please don't do that in this PR)